src/Identity/samples/IdentitySample.Mvc/Startup.cs

```diff
@@ -53,6 +55,8 @@ public void ConfigureServices(IServiceCollection services)
             // Add application services.
             services.AddTransient<IEmailSender, AuthMessageSender>();
             services.AddTransient<ISmsSender, AuthMessageSender>();
+
+            services.AddSingleton<IDeveloperPageExceptionFilter, DatabaseExceptionFilter>();
```

Comment:
Do you think we need a convenience API for this?

`services.AddDatabaseExceptionPageSupport();`

Also, does this need `TryAddEnumerable`?

Reply:
Sure this actually simplifies the using statements too.